tdrs-backend/.env.example

```diff
@@ -1,43 +1,74 @@
 ### Local Development Environment
 # Copy this file to `.env` and replace variables as needed
 #
-DISABLE_COLLECTSTATIC=1
 
-# Parameters defined in the login.gov documentation https://developers.login.gov/oidc/
-CLIENT_ID=urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-dev
-OIDC_RP_CLIENT_ID=urn:gov:gsa:openidconnect.profiles:sp:sso:hhs:tanf-proto-dev
-ACR_VALUES=http://idmanagement.gov/ns/assurance/ial/1
-OIDC_OP_ISSUER=https://idp.int.identitysandbox.gov/
-OIDC_OP_JWKS_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/certs
-OIDC_OP_AUTHORIZATION_ENDPOINT=https://idp.int.identitysandbox.gov/openid_connect/authorize
-OIDC_OP_LOGOUT_ENDPOINT=https://idp.int.identitysandbox.gov/openid_connect/logout
-OIDC_OP_TOKEN_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/token
-OIDC_OP_USER_ENDPOINT=https://idp.int.identitysandbox.gov/api/openid_connect/userinfo
-CLIENT_ASSERTION_TYPE=urn:ietf:params:oauth:client-assertion-type:jwt-bearer
+###
```

Comment:
Did some cleanup in this file to organize and better document options. The idea is that this will hold an exhaustive list of all environment variables possible for the backend.